### PR TITLE
Quickstart Guides: Adding separators

### PR DESCRIPTION
**Pull request by rickiesherman**
screen shot 2017-04-14 at 13 03 55
I added separators to the end of each section. Looks much more readable!
Left is new, right is old.

### CODE REVIEW

File: repositories/quickstarts/_posts/2016-12-12-nodejs-quickstart.md [hunk 1]
```diff
@@ -21,23 +23,29 @@ This guide will help you setup Runnable templates for a simple 2-tier Node.js ap
...
+
+---
 
-### Packages 
+### Packages
 
 This option allows you to install any tools, libraries, or frameworks you need. Add multiple packages by separating them with spaces. You can specify packages that are listed in the Ubuntu Aptitude library.
   ![node4]({{ site.baseurl }}/images/node4.png)  
 
+  ---
```
Comment (taylordolan) [inline repositories/quickstarts/_posts/2016-12-12-nodejs-quickstart.md:35]: It doesn't affect the output but you should remove the indentation.

File: repositories/quickstarts/_posts/2016-12-12-php-quickstart.md [hunk 2]
```diff
@@ -10,16 +10,22 @@ description: How to setup your multi-tier PHP application.
...
 3. Follow the suggested recommendations for build and start commands. See below for slight modifications.
 
+---
+
 #### Set proper permissions for your application directory
   1. Add the following lines to the **Build Commands** section under *Commands & Packages*
   `$ chgrp -R www-data /var/www/html/`
   `$ chmod -R 775 /var/www/html/storage`
 
+  ---
```
Comment (taylordolan) [inline repositories/quickstarts/_posts/2016-12-12-php-quickstart.md]: Remove indentation

File: repositories/quickstarts/_posts/2016-12-12-php-quickstart.md [hunk 3]
```diff
@@ -44,8 +50,12 @@ In order to get a Laravel app running, here are the following steps you need to
...
 
 4. Add the following scripts to Scripts
-  ```$ a2ensite laravel.conf
-  $ a2enmod rewrite```
+  ```
+  $ a2ensite laravel.conf
+  $ a2enmod rewrite
+  ```
+
+  ---
```
Comment (taylordolan) [inline repositories/quickstarts/_posts/2016-12-12-php-quickstart.md]: Remove indentation

Comment (rickiesherman) [reply repositories/quickstarts/_posts/2016-12-12-php-quickstart.md]: fixed this up

**Review by taylordolan — APPROVED** [review]

**Pull request merged**